src/components/Greeting.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { getRandomGreeting } from '../actions/greetingActions';
+
+const Greeting = ({ greeting, getRandomGreeting }) => {
+  useEffect(() => {
+    getRandomGreeting();
+  }, [getRandomGreeting]);
+
+  return (
+    <div>
+      <h1>{greeting}</h1>
+    </div>
+  );
+};
+
+Greeting.propTypes = {
+  greeting: PropTypes.string.isRequired,
+  getRandomGreeting: PropTypes.func.isRequired,
+};
+
+const mapStateToProps = (state) => ({
+  greeting: state.greeting,
+});
+
+export default connect(mapStateToProps, { getRandomGreeting })(Greeting);
```

Comment:
- [ ] for some weird reasons your frontend app is showing blank, kindly fix
![image](https://user-images.githubusercontent.com/66097371/226101525-e5eaab0f-8f37-4ccc-904c-2ced24077301.png)